core/primitives/src/types.rs

```diff
@@ -119,13 +119,35 @@ pub struct FunctionCallTransaction {
     pub args: Vec<u8>,
 }
 
+#[derive(Hash, Serialize, Deserialize, PartialEq, Eq, Debug, Clone)]
+pub struct CreateAccountTransaction {
+    pub nonce: u64,
+    pub sender: AccountId,
+    // receiver is the account id to be created
```

Comment:
these structs exist so we don't have to have comments like this. `new_account_id` works